.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            4
+        ],
+        "quotes": [
+            2,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ]
+    },
+    "env": {
+        "es6": true,
+        "node": true,
+        "browser": true
+    },
+    "extends": "eslint:recommended"
+}
```

Comment:
yeah, I was gonna create an issue for you to set us up a .editorconfig.

that being said this file is generated by eslint, I do think we will have rule changes to make as we go - this is only a baseline.